magefiles/ci.go

```diff
@@ -18,7 +21,14 @@ func ciSetup() error {
 
 	mg.Deps(CheckForPulsarRunning)
 
-	err = dockerComposeRun("up", "-d", "server", "executor")
+	// By starting the executor first,
+	// we can ensure that the server will be able to connect to it
```

Comment:
the server does not connect to the executor -- but maybe this buys a little time for pulsar init?

Reply:
oh i see, it needs to receive poll from executor or job submit fails.

Reply:
I should probably make this comment more clear.

Essentially there is a "executor check" that the server runs, to see what executor clusters are available. The executor can be sending information to the server, but this check can still fail.

This happens on a poll everything 1 minute I think.